05-upgrades-management/code/scripts/upgrade.js

```diff
@@ -0,0 +1,17 @@
+// scripts/upgrade.js
+const { ethers } = require("hardhat");
+
+async function main () {
+  const newImplName = 'BoxV2';
+  const NewImpl = await ethers.getContractFactory(newImplName);
+  console.log(`Upgrading to ${newImplName}...`);
+  await upgrades.upgradeProxy('replace with your proxy address', NewImpl);
```

Comment:
I'd move this to a `const proxyAddress = 'replace with...'` as you did above, so it's easier not to miss!